lbrynet/daemon/Daemon.py

```diff
@@ -2517,7 +2517,6 @@ def jsonrpc_block_show(self, blockhash=None, height=None):
             d = self.wallet.get_block(blockhash)
         elif height is not None:
             d = self.wallet.get_block_info(height)
-            d.addCallback(lambda b: self.wallet.get_block(b))
         else:
             # TODO: return a useful error message
             return server.failure
```

Comment:
You could refactor this whole function to just `return` `get_block()` and `get_block_info()`, `self._render_response()` doesn't do anything.